### DISCUSSION TITLE
[Project (beta)] How to remove a project

### DISCUSSION THREAD

**Discussion by winsphinx** [community/community#6355] [Projects and Issues]
How to remove a project in beta version?

**Comment by prin53** [comment]
There is an option to close a project in settings (a gear in the corner):

<img width="1165" alt="Screenshot" src="https://user-images.githubusercontent.com/4447833/137452114-25223e8f-008e-4d33-9874-272444c56a81.png">

**Comment by winsphinx** [comment]
Thank you, eh...but that's only a close. I'd like to remove it permantly, as the lagacy one.
